src/cli/impl/steps/UpdateStep.cpp

```diff
@@ -6,10 +6,20 @@
 
 StepResult UpdateStep::Execute(Context &context)
```

Comment:
better name can be chosen, not clear what it updates, as it reads label, generates command and clears context